src/olympia/reviewers/tests/test_serializers.py

```diff
@@ -33,18 +34,39 @@ def setUp(self):
         extract_version_to_git(self.addon.current_version.pk)
         self.addon.current_version.refresh_from_db()
 
-    def get_serializer(self, obj, **extra_context):
+        extract_version_to_git(self.addon.current_version.pk)
+        self.addon.current_version.reload()
+        assert (self.addon.current_version.current_file.filename ==
+                'notify-link-clicks-i18n.xpi')
+        self.version = self.addon.current_version
+        self.file = self.addon.current_version.current_file
+
+    def get_serializer(self, **extra_context):
         api_version = api_settings.DEFAULT_VERSION
         request = APIRequestFactory().get('/api/%s/' % api_version)
         request.versioning_scheme = api_settings.DEFAULT_VERSIONING_CLASS()
         request.version = api_version
         extra_context.setdefault('request', request)
```

Comment:
I have a question about this code, the code that creates the request, configures it, and passes it into `extra_context`. It doesn't seem to be needed by the code (which doesn't access anything in `request`), and if I remove it from the test cases they all still pass. Is this something that is needed for each of these tests, and if so, why?

Reply:
Looks like it was needed when `download_url` was added in 1a7217dcfad9514ce23ed00a0aaba94e82dd9272 because back then the download view was in DRF and we used `reverse()` from DRF which needs a `request`, but it was later changed in e67eeccaf848b0fc2eaa4304bd2b041bf84a9551 to call the "regular" download view which is not an API, so we removed the need for DRF's `reverse', which in turn meant we no longer need to rely on `request` from the serializer context.

DRF still provides `request` when it initializes a serializer from a view, but we can forget about all this and remove that code if we no longer depend on it.

Reply:
Ok, so we can remove it from all the tests in this file?

Reply:
If tests pass, yes